### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by ericagates**
Create a Side Bar Component to be called in the view/edit/create page

### CODE REVIEW

File: pet_wellness_app1/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -25,10 +25,29 @@
      background-color: lightgrey;
  }
 
+<<<<<<< HEAD
```
Comment (berkeleycole) [inline pet_wellness_app1/app/assets/stylesheets/application.scss]: Hmm, looks like there was a bad merge here, you'll want to remove these head references and make sure the code is what you want

File: pet_wellness_app1/app/javascript/components/pages/AddPet.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 import React, { Component } from 'react'
```
Comment (berkeleycole) [inline pet_wellness_app1/app/javascript/components/pages/AddPet.js]:
```suggestion
import React, { Component, Fragment } from 'react'
```

File: pet_wellness_app1/app/javascript/components/pages/AddPet.js [hunk 3]
```diff
@@ -11,7 +18,15 @@ class AddPet extends Component{
     render(){
         return(
             <React.Fragment>
```
Comment (berkeleycole) [inline pet_wellness_app1/app/javascript/components/pages/AddPet.js]:
```suggestion
            <Fragment>
```

**Review by berkeleycole — CHANGES_REQUESTED** [review]
I'd handle the merge conflict tags before merging in order to not get any of that into the master branch, ping me if you have any questions about it!

**Review by jakobbergeson — APPROVED** [review]

File: pet_wellness_app1/app/assets/stylesheets/application.scss (hunk 1 above)
Comment (jakobbergeson) [reply pet_wellness_app1/app/assets/stylesheets/application.scss]: Okay good catch! We fixed the conflict and are ready to merge!

**Review by berkeleycole — APPROVED** [review]

**Pull request merged**